core/src/nucleus/mod.rs

```diff
@@ -429,7 +440,8 @@ pub fn reduce(
 #[cfg(test)]
 mod tests {
     use super::{
-        super::{nucleus::Action::*, state::Action::*}, *,
+        super::{nucleus::Action::*, state::Action::*},
+        *,
     };
```

Comment:
could you make this import absolute and also add a `use super::*;` above?

Reply:
ok done.